geo-types/Cargo.toml

```diff
@@ -15,6 +15,7 @@ use-rstar = ["rstar", "approx"]
 
 [dependencies]
 approx = { version = "0.4.0", optional = true }
+arbitrary = { version = "1.0.0-rc1", optional = true }
```

Comment:
Do you know how pre-release prefixes work with semver?

Are breaking changes allowed between a release candidate and the eventual release?  In other words, if we merge this as is, does semver require us to do a breaking rev of geo-types? 😬 

I tried looking for a little bit and haven't yet found an answer...

Reply:
I dunno what semver's official policy is but my guess is yes it would be considered a breaking change. I thought about what you said as I opened this pull request and my feeling is that since it's very unlikely anyone will use this, and that the breakage between 1.0.0-rc1 and 1.0.0 is going to be small, that we should lie a little and just upgrade to 1.0.0 in a geo-types minor release 🤷🏻 

If people feel otherwise we can wait until 1.0.0 is released https://github.com/rust-fuzz/arbitrary/issues/62

Reply:
<div class="tenor-gif-embed" data-postid="6177343" data-share-method="host" data-width="100%" data-aspect-ratio="1.7785714285714287"><a href="https://tenor.com/view/beavis-butthead-breakin-the-law-gif-6177343">Beavis And Butthead Breakin GIF</a> from <a href="https://tenor.com/search/beavisandbutthead-gifs">Beavisandbutthead GIFs</a></div><script type="text/javascript" async src="https://tenor.com/embed.js"></script>

Reply:
Ok, but if the SemVer Police show up, I'm claiming I was coerced.